### ISSUE TITLE
Redirection error

### ISSUE THREAD

**Issue by sunman4008** [johngodley/redirection#1495]
Hello,

I am redirecting everything from old website www.ichimokutrade.com to new website www.imtftrade.com.  

https://www.ichimokutrade.com/c/wp-content/uploads/2017/06/2017-06-03_141959.xlkxlfpng.png

and i want it going to

https://www.imtftrade.com/wp-content/uploads/2017/06/2017-06-03_141959.xlkxlfpng.png

I have setup the following:  ( shared attachment)
![2019-02-21_090559redirection](https://user-images.githubusercontent.com/19915128/53174763-90235500-35b8-11e9-9017-63b24a644423.jpg)

https://1drv.ms/u/s!An4NvA_KUbEfgZs5rV7MSYIgrwF9Uw

It is not working when I put https://www.ichimokutrade.com/c/wp-content/uploads/2017/06/2017-06-03_141959.xlkxlfpng.png  in a browser, I get a 404 error.   

Manesh

**Comment by johngodley** [comment]
Your redirect is for `/c/wp-content/uploads`. That does not seem to match the image URL.

**Comment by sunman4008** [comment]
the matched target is where I want it to go?   Is that correct?

**Comment by sunman4008** [comment]
I have all sub directories in this location.    So I am trying to not do a redirection for each file.

**Comment by johngodley** [comment]
Sure, but the source URL needs to match the original URL. `/c/wp-content/uploads` does not match `/c/wp-content/uploads/2017/06/2017-06-03_141959.xlkxlfpng.png` - it just matches `/c/wp-content/uploads`

You can use a [regular expression](https://redirection.me/support/redirect-regular-expressions/) to match multiple sources.

Also, as you are doing a redirect from another domain then that domain will need to be mapped to the same WordPress site the redirect is created on.

**Comment by sunman4008** [comment]
Hello,

 

I did the following and still not working:

 



 

 



Thank you,

 

 

Manesh Patel, CMT, CFTe

IchimokuTrade

"Any Instrument, Any Timeframe, and No Expiration"

 

Tel:    (678) 389-9053

Fax:    (678) 264-0910 

Cell:   (678) 453-8233 

Email:  mpatel@eiicapital.com <mailto:mpatel@eiicapital.com> 

SKYPE: mpatel00001

        

 <http://www.ichimokutrade.com/> 

  <https://www.ichimokutrade.com/c/blog>    <http://www.facebook.com/Ichitrader>     <http://www.twitter.com/ichimokutrading>     <https://www.youtube.com/user/sunman4008/featured>     <https://www.linkedin.com/pub/manesh-patel/2/a08/a85>     <https://plus.google.com/113408722847099007978/posts> 

 

CONFIDENTIALITY NOTICE:  This email is intended only for the person or entity to 

which it is addressed and may contain confidential and/or privileged material. 

Delivery of this email or any of the information contained herein to anyone other

than the intended recipient or his designated representative is unauthorized and 

any other use, reproduction, distribution or copying of this document or the 

information contained herein, in whole or in part, without the prior written 

consent of sender or its affiliates is prohibited and may be unlawful. 

Any performance information contained herein may be unaudited and estimated. 

Past performance is not necessarily an indication of future performance. 

If you have received this message in error, please notify the sender immediately 

and delete this message and any related attachments.  

Commity Future Trading Commission, Forex, Futures, Equity and options trading 

has large potential rewards, but also large potential risk.

**Comment by johngodley** [comment]
If you included an image it is not visible on Github

**Comment by sunman4008** [comment]
![2019-02-25_103344newriected](https://user-images.githubusercontent.com/19915128/53348361-ea8a2180-38e8-11e9-884a-0ec0c73595ae.jpg)


Sorry...here is the image

**Comment by johngodley** [comment]
Ok, I'm afraid that's still not a valid regular expression. You can use https://regex101.com/ to test an expression. In your image above you should remove the `^/` after the `uploads/` bit.

**Comment by sunman4008** [comment]
Sorry...I am a novice at this.   In your tool, what I put in the regular expression?  I am assuming the regix which I thought would be https://www.ichimokutrade.com/c/wp-content/uploads/(.*)   but does not like the / character.    Is there a video tutorial you have on the tool so I don't keep bothering you.

**Comment by johngodley** [comment]
It's not my tool, just a general purpose website for regular expressions. Press the vertical ... to change the delimiter to something that is not `/`.

**Comment by sunman4008** [comment]
Hello,

I got that problem figure out.   I got one more issue.   I have a link http://blog.eiicapital.com/wp-content/uploads/2008/01/gd_1_24_2008.jpg.     The blog.eiicapital.com is not a domain.  It was a a directory in the server.    Therefore, if I try to do a redirect to this....it says Ip address not found.   Is there a way to do a redirect on a file location so if my blog is referencing the file, it redirect it?
